polus-vector-to-label-plugin/src/main.py

```diff
@@ -0,0 +1,323 @@
+from os import cpu_count
+from bfio import BioReader, BioWriter, OmeXml
+import argparse, logging
+import numpy as np
+from pathlib import Path
+from cellpose import dynamics, utils
+import torch
+from concurrent.futures import ThreadPoolExecutor, wait, Future
+import typing
+
+""" Plugin Constants """
+TILE_SIZE = 2048    # Largest chunk of an image to process
+TILE_OVERLAP = 256  # Amount of overlap between tiles
+NITER = 200         # Number of iterations to run flow dynamics
+
+# Use a gpu if it's available
+USE_GPU = torch.cuda.is_available()
+if USE_GPU:
+    DEV = torch.device("cuda")
+else:
+    DEV = torch.device("cpu")
+
+# Initialize the logger
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("main")
+logger.setLevel(logging.INFO)
+
+def overlap(previous_values: np.ndarray,
+            current_values: np.ndarray,
+            tile: np.ndarray
+            ) -> np.ndarray:
+    """Resolve label values between tiles
+    
+    This function takes a row/column from the previous tile and a row/column
+    from the current tile and finds labels that that likely match. If labels
+    in the current tile should be replaced with labels from the previous tile,
+    the pixels in the current tile are removed from ``tile`` and the label value
+    and pixel coordinates of the label are stored in ``labels`` and ``indices``
+    respectively.
+
+    Args:
+        previous_values (np.ndarray): Previous tile edge values
+        current_values (np.ndarray): Current tile edge values
+        tile (np.ndarray): Current tile pixel values, flattened
+
+    Returns:
+        np.ndarray: Current tile pixel values, flattened
```

Comment:
you actually return three objects, not one. Type hinting and docstring should be updated.